inst-hooks/03-disable-ztd-file.pre-upgrade.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# Ensure that we don't enter ZTD mode after an upgrade
+
+# Deleting the postinst hook ensures that even a failure in a later hook
+# would not leave the ZTD file hanging around on the standby filesystem.
+rm -f /alt/root/hooks/95-enable-ztd-mode.postinst.sh
```

Comment:
Is this directory created when ztd runs? Can you leave a comment regarding that? I don't see any `/alt` directory in the rootfs.